Sources/BSWInterfaceKit/Views/PhotoGalleryView.swift

```diff
@@ -57,11 +57,7 @@ final public class PhotoGalleryView: UIView {
     public weak var delegate: PhotoGalleryViewDelegate?
     
     /// Enables or disables the zoom for the photos.
-    public var zoomEnabled = false {
-        didSet {
-            collectionView.reloadData()
```

Comment:
Is this necessary to delete?

Reply:
It can be deleted because, as it is now private, it can only be set in init